doc/releases.md

```diff
@@ -568,6 +568,8 @@ To ensure communication goes out with the timing of the blog post, please allow
 will be shared with the community in the email to coordinate these
 announcements.
 
+Ping the IRC ops and the other [Partner Communities] liaisons.
```

Comment:
Are there any handles? Also, not sure which IRC channel this would be..#node-build?

Reply:
Probably `#node-dev`, as for handles, if they are not online they won't see the notification, so I'd say mention those who are online, e.g.:
![image](https://user-images.githubusercontent.com/96947/47956300-c0e36e00-df78-11e8-9017-7b831aeb0924.png)

Reply:
Even just a message in #node-build and/or #node-dev would be immensely helpful.

Reply:
```suggestion
Mention the new release in [freenode#node-build], this should sync with our other [Partner Communities] and inform our liaisons.
[freenode#node-build]: https://www.irccloud.com/irc/freenode/channel/node-build
```

Reply:
This seems like something we should automate rather than do manually

Reply:
Like 90% of the things in this guide that are still manual 😅